app/presenters/offender_with_prison_timeline_presenter.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class OffenderWithPrisonTimelinePresenter
```

Comment:
I've got no opinion about most of the code, seems well done.

The only comment I've got is about the OffenderWithTimelinePresenter - I'm actively trying to move us away from offender wrappers and am a little concerned we've added another one to the mix. The whole offender wrapper anti-pattern takes us away from proper OO coding standards, and I'd suggest we start moving back to proper OO.

Why not have a lazy initialized `timeline` method on MpcOffender or AllocatedOffender, which instantiates a PrisonerTimeline object when first called?

`def timeline; @prisoner_timeline ||= PrisonerTimeline.new(self); end`

Happy to discuss it further if I'm missing something that requires more explanation

Reply:
(oh, and p.s.: you don't need the frozen string literal comment any more on modern Rubies)

Reply:
I'd also agree with this point. I think we should be moving away from wrappers around MpcOffender, so if we can find a way that we don't need to create a new one, that'd be grand!

Reply:
Sounds like a plan - lets discuss when I come back to this